index.js

```diff
@@ -85,7 +85,8 @@ module.exports = (urlString, opts) => {
 		urlObj.hostname = urlObj.hostname.replace(/\.$/, '');
 
 		// Remove `www.`
-		if (opts.stripWWW) {
+		// eslint-disable-next-line no-useless-escape
+		if (opts.stripWWW && /^www\.([a-z\-0-9]{2,63})\.([a-z\.]{2,5})$/.test(urlObj.hostname)) {
```

Comment:
`0-9` => `\d`

Reply:
Can you add a short comment about what the `2,63` and `2,5` numbers mean?

Reply:
Yeah, but first I need to get some sleep :) (IMO it's obvious it represents min and max length :P but yeah, in case someone doesn't know ;) )

Reply:
I don't mean what they mean in regex, I mean, for example, why it's 63 in length.

Reply:
I'm very lazy :P https://en.m.wikipedia.org/wiki/Hostname#Restrictions_on_valid_host_names

Reply:
This has no international support.